students/jiyeongkim/westagram/users/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+# Create your models here.
+class User(models.Model):
+    name            = models.TextField(max_length=45)
+    email           = models.CharField(max_length=45, unique=True)
+    Password        = models.CharField(max_length=45, null=False)
+    line            = models.IntegerField(unique=True)
```

Comment:
- 변수명(필드명)은 소문자로 해주세요.
- 연락처의 의미로 line이라는 변수명을 사용했다면 해당 필드는 IntegerField가 아닌 CharField로 변경해 주세요. 연락처는 숫자보다는 문자의 의미가 더 강합니다.
- email field에 unique 속성을 추가하셔서 유저가 중복 저장되는 문제가 발생하지 않도록 수정해주세요!(공식 문서에서 그 방법을 찾아보고 적용해주세요.)
- 이름과 관련된 필드는 래를 참고해서 수정해 주세요.
    - id(계정) → `username`
    - name → `first_name`, `last_name`
 - =을 기준으로 정렬해 주세요.

```suggestion
class User(models.Model):
    name     = models.TextField(max_length=45)
    email    = models.CharField(max_length=45, unique=True)
    password = models.CharField(max_length=45, null=False)
    line     = models.IntegerField(unique=True)
```